open-search/oci/opensearch-upload.py

```
@@ -0,0 +1,46 @@
+                result[key] = value
+    title=file_name.strip().split(".", 1)[0]
+    all[title]=result
+
+custom_timestamp = datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
+all["timestamp"]=custom_timestamp
+all["attributes"]=args.attributes
+all["category"]=args.category
+json_str = json.dumps(all, indent=2)
+#print(json_str)
+
+OPENSEARCH_URL = os.environ["url"]
```

Ruff (0.12.2)

Use capitalized environment variable `URL` instead of `url`

Replace `url` with `URL`

(SIM112)